### PR TITLE
restock() proc

### LINKED ISSUE #7112: Medical Cyborg Syringe
If you switch to harm intent and jab someone with a syringe, it breaks and you have to find a new one.

If you do it with the mediborg one, it also breaks.

You cannot fix the syringe for the mediborg via charging or any other means other than getting a module reset.

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #7112

also a bunch of absolute pathing where there was a ton of relative pathing

### CODE REVIEW

File: code/game/machinery/rechargestation.dm [hunk 1]
```diff
@@ -308,6 +308,11 @@
 					if(istype(O,/obj/item/weapon/melee/defibrillator))
 						var/obj/item/weapon/melee/defibrillator/D = O
 						D.charges = initial(D.charges)
+					if(istype(O,/obj/item/weapon/reagent_containers/syringe))
```
Comment (Intigracy) [inline code/game/machinery/rechargestation.dm]: But this is a proc to handle recharging.

Comment (Intigracy) [inline code/game/machinery/rechargestation.dm]: And all of this is the recharge proc.

Comment (clusterfack) [inline code/game/machinery/rechargestation.dm]: https://en.wikipedia.org/wiki/Object-oriented_programming
https://en.wikipedia.org/wiki/Polymorphism_(computer_science)

i.e. atom/proc/recharge_borg_module()
```
for(object/A in modules)
    A.recharge_borg_module()
```

Comment (Intigracy) [inline code/game/machinery/rechargestation.dm]: ah, you mean do the proc on individual items instead of a big list of them all.

Comment (Intigracy) [inline code/game/machinery/rechargestation.dm]: ¯_(ツ)_/¯ maybe someday

Comment (clusterfack) [inline code/game/machinery/rechargestation.dm]: Fix it, it'll just keep getting longer this way (which I figured it would a year ago in the quoted post)

Comment (Intigracy) [inline code/game/machinery/rechargestation.dm]: You're a cruel mistress indeed.

**Comment by Intigracy** [comment]
@clusterfack ye be a cruel mistress indeed

**Pull request merged**